src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf?) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf) format("ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc) format("ttc");
+}
+
+* {
+  font-family: "Avenir-Book", sans-serif;
+}
+
+.search__big {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big__input {
+  transition: outline 0.5s;
+  width: 100%;
+  padding: 23px 62px;
+  font-size: 16px;
+}
+
+input:hover {
+  outline: solid 2px rgb(17, 201, 183);
+}
+
+input:focus {
+  outline: solid 2px purple;
+  font-weight: bold;
+}
```

Comment:
do not use tags to style elements. you have classes for these inputs (`small__input` and `big__input`) use them.